# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/xul/nsBoxFrame.cpp
# Commit: a2da895a58ce
# Full Hash: a2da895a58ce094d55e6a2e040dd648962b39868
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2022-02-22 09:37:09
# Regressor Bug: 1714584
# File Overlap Count: 2
# Description:
#   Bug 1714584 - Part 1: Decouple nsDisplayList internal list from nsDisplayItems r=mstange
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138152
# ==============================================================================

diff -r c3b3315d2344 -r a2da895a58ce layout/xul/nsBoxFrame.cpp
--- a/layout/xul/nsBoxFrame.cpp	Mon Feb 21 21:57:03 2022 +0000
+++ b/layout/xul/nsBoxFrame.cpp	Tue Feb 22 00:44:25 2022 +0000
@@ -961,7 +961,7 @@
     // and merge them into a single Content() list. This can cause us
     // to violate CSS stacking order, but forceLayer is a magic
     // XUL-only extension anyway.
-    nsDisplayList masterList;
+    nsDisplayList masterList(aBuilder);
     masterList.AppendToTop(tempLists.BorderBackground());
     masterList.AppendToTop(tempLists.BlockBorderBackgrounds());
     masterList.AppendToTop(tempLists.Floats());